params.yaml

```diff
@@ -439,3 +439,15 @@ ratio_study:
 export:
   triad_code: "3"
   run_id: "2023-03-14-clever-damani"
+
+
+# Comp -------------------------------------------------------------------------
+
+comp:
+  # Number of comps to generate for each card.
+  # Only used if comp_enable = TRUE
+  num_comps: 20
+  # Number of price bins to use when binning properties for the purpose of
+  # comparison. Corresponds to Ntile predicted FMV, e.g. 10 creates deciles
+  # of predicted values
+  num_comp_price_bins: 30
```

Comment:
I don't know how I feel about adding an entirely new top-level param exclusively for comps, but I couldn't think of another top-level key where it would make sense to store these. Got any ideas?

Reply:
I think this is fine! However, I would put the `comp` key before the `export` key, since the keys are ~ roughly ~ in order of the pipeline steps that use them.

Reply:
Makes sense, done in e4ec5a383621fc51b49169a8017c5082bcde43fd!